test/ctia/task/migrate_es_stores_test.clj

```diff
@@ -0,0 +1,210 @@
+             [core :as helpers :refer [post with-atom-logger]]
+             [es :as es-helpers]
+             [fake-whoami-service :as whoami-helpers]]
+            [ctim.domain.id :refer [make-transient-id]]
+            [ctim.examples
+             [actors :refer [actor-minimal]]
+             [campaigns :refer [campaign-minimal]]
+             [coas :refer [coa-minimal]]
+             [exploit-targets :refer [exploit-target-minimal]]
+             [incidents :refer [incident-minimal]]
+             [indicators :refer [indicator-minimal]]
+             [judgements :refer [judgement-minimal]]
+             [relationships :refer [relationship-minimal]]
+             [sightings :refer [sighting-minimal]]
+             [ttps :refer [ttp-minimal]]]))
+
+(use-fixtures :once
+  (join-fixtures [mth/fixture-schema-validation
+                  helpers/fixture-properties:clean
+                  es-helpers/fixture-properties:es-store
+                  helpers/fixture-ctia
+                  whoami-helpers/fixture-server
+                  es-helpers/fixture-delete-store-indexes]))
+
+(use-fixtures :each
+  whoami-helpers/fixture-reset-state)
+
+(def fixtures-nb 100)
+
+(defn post-bulk [examples]
+  (let [{status :status
+         bulk-res :parsed-body}
+        (post "ctia/bulk"
+              :body examples
+              :headers {"Authorization" "45c1f5e3f05d0"})]
+    (is (= 201 status))))
+
+(defn randomize [doc]
+  (assoc doc
+         :id (make-transient-id "_")))
+
+(defn n-doc [fixture nb]
+  (map randomize (repeat nb fixture)))
+
+(defn refresh-index [host port index]
+  (let [index-refresh-url
+        (format "http://%s:%s/%s/_refresh" host port index)]
+    (:status (client/post index-refresh-url))))
+
+(def examples
+  {:actors (n-doc actor-minimal fixtures-nb)
+   :campaigns (n-doc campaign-minimal fixtures-nb)
+   :coas (n-doc coa-minimal fixtures-nb)
+   :exploit-targets (n-doc exploit-target-minimal fixtures-nb)
+   :incidents (n-doc incident-minimal fixtures-nb)
+   :indicators (n-doc indicator-minimal fixtures-nb)
+   :judgements (n-doc judgement-minimal fixtures-nb)
+   :relationships (n-doc relationship-minimal fixtures-nb)
+   :sightings (n-doc sighting-minimal fixtures-nb)
+   :ttps (n-doc ttp-minimal fixtures-nb)})
+
+(deftest add-groups-test
+  (is (= (transduce sut/add-groups conj [{}])
+         [{:groups ["tenzin"]}]))
+  (is (= (transduce sut/add-groups conj [{:groups []}])
+         [{:groups ["tenzin"]}]))
+  (is (= (transduce sut/add-groups conj [{:groups ["foo"]}])
+         [{:groups ["foo"]}])))
+
+(deftest fix-end-time-test
+  (is (= (transduce sut/fix-end-time conj [{}]) [{}]))
+  (is (= (transduce sut/fix-end-time conj [{:valid_time
+                                            {:start_time "foo"}}])
+         [{:valid_time
+           {:start_time "foo"}}]))
+  (is (= (transduce sut/fix-end-time conj
+                    [{:valid_time
+                      {:end_time "2535"}}])
```

Comment:
I think the test should use a more realistic `#inst` here instead of the string "2535".